shoop/admin/modules/contacts/views/detail.py

```diff
@@ -58,6 +62,26 @@ def build_new_user_button(self):
             extra_css_class="btn-gray btn-inverse",
         ))
 
+    def build_deactivate_button(self):
+        if not self.contact.is_active:
```

Comment:
This if else is not cool. Do it with help function or change the value/text based on the contact.is_active